gofakes3_test.go

```diff
@@ -1300,6 +1300,189 @@ func TestListBucketWithStorageClass(t *testing.T) {
 	}
 }
 
+func TestCreateObjectWithStorageClassStandard(t *testing.T) {
+	tempDir, err := ioutil.TempDir("", "s3bolt")
+	if err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
it should not be necessary to create temp directory just to create a temporary file.

Reply:
I'm going to merge the PR as is. Thanks.